src/matcha_ml/cli/analytics.py

```diff
@@ -0,0 +1,24 @@
+"""Matcha CLI for opting out/in of collecting analytics."""
+import typer
+from rich import print
+
+from matcha_ml.services.global_parameters_service import GlobalParameters
+
+# create a typer app to group all analytics subcommands
+app = typer.Typer()
+
+
+@app.command()
+def opt_out() -> None:
+    """Opt-out of collecting analytics."""
+    print("We respect your privacy. No analytics data will be collected.")
+    return GlobalParameters().opt_out_of_analytics()
+
+
+@app.command()
+def opt_in() -> None:
+    """Opt-in for collecting analytics."""
+    print(
+        "Thank you for allowing matcha to collect analytics data. This is very helpful!"
```

Comment:
Thank you for enabling data collection, this helps us improve matcha and anonymously understand how people are using the tool.